cheevos/cheevos.c

```diff
@@ -264,9 +264,8 @@ static void rcheevos_activate_achievements(void)
          {
             char buffer[256];
             buffer[0] = '\0';
-            /* TODO/FIXME - localize */
             snprintf(buffer, sizeof(buffer),
-               "Could not activate achievement %u \"%s\": %s",
+               msg_hash_to_str(MSG_CHEEVOS_COULD_NOT_ACTIVATE_ACHIEVEMENT),
                achievement->id, achievement->title, rc_error_str(result));
```

Comment:
I'm not sure it's reasonable to localize this as `rc_error_str` will still return an English error message.